documentation/modules/oauth/proc-oauth-authorization-broker-config.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following module:
+//
+// con-oauth-config.adoc
+
+[id='proc-oauth-authorization-broker-config-{context}']
+= Configuring {oauth} authorization support
+
+This procedure describes how to configure Kafka brokers to use {oauth} authorization using {oauth-server} Authorization Services.
+
+The properties used to configure authorization for Kafka brokers might already be configured for xref:proc-oauth-authentication-broker-config-{context}[{oauth} authentication].
+If so, they are also used for authorization if the equivalent authorization properties are not defined.
```

Comment:
That's not true. If you configure authentication using `oauth` you have to configure the authorization part independently; it cannot use the authentication parameters.

Reply:
Thanks @ppatierno. Marko has clarified in a later comment. And suggests we remove these sentences.

Reply:
Removed